incubator/jenkins/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
AFAIK, we would need to add this to helm as a part of the "helm create" template. @prydonius @michelleN @technosophos .